src/PoolManager.sol

```diff
@@ -52,42 +52,6 @@ contract PoolManager is IPoolManager, ProtocolFees, NoDelegateCall, ERC6909Claim
         return pools[id];
     }
 
```

Comment:
shall we make `mapping(PoolId id => Pool.State) public pools;` internal to save there being a getter generated?